layouts/partials/redirect.html

```diff
@@ -0,0 +1,12 @@
+{{ if .Params.redirect_to }}
+  {{ $redirect_to := .Params.redirect_to }}
+  {{ $redirect_after := .Params.redirect_after | default "0" }}
```

Comment:
Do we want a default of 0 here? Or do we want a default of something like 1? I only ask because it might be beneficial to let it sit for at least a second.

Reply:
I don't feel strongly either way, but I am leaning towards 0 as a default because I expect redirects to be immediate, unless there is a good reason to let the page flash for a second.

Reply:
Well, I'm thinking about the use case of redirecting to other external things as a tool (and that's good because people aren't just jolted out of the site suddenly); I feel like the only case where it might be used to redirect internally would be `/jobs/` -> `/`, so that's the minority case.

Reply:
I think for the purpose of simply not exposing `/jobs/`, we should keep this as a fast internal redirect. Perhaps we can make changes in the future to repurpose this into something that is meant to handle external links, maybe with better copy and UI like adding animation to countdown the timer.